config/yugabyte/regression_pipelines/scan_workloads/yugabyte/scanG12_bitmapscan_orCondition.yaml

```diff
@@ -0,0 +1,117 @@
+#TEST FOR YUGABYTE REGULAR TABLES
+#GOAL12 from tab Scan queries of doc https://docs.google.com/spreadsheets/d/1-M1i5tWJQJdbmtv7mz_20MZsT2Oqh_T3-6m12qKAMcA/edit#gid=796784261:
+# Bitmap scan on pkey, hash, and range columns using an OR condition - Varying number of rows returned
+#TESTS:
+#   Bitmap scan time to fetch 10 rows
+#   Bitmap scan time to fetch 100 rows
+#   Bitmap scan time to fetch 1000 rows
+#   Bitmap scan time to fetch 10000 rows
+type: YUGABYTE
+driver: com.yugabyte.Driver
+url: jdbc:yugabytedb://{{endpoint}}:5433/yugabyte?sslmode=require&ApplicationName=featurebench&reWriteBatchedInserts=true&load-balance=false
+username: {{username}}
+password: {{password}}
+batchsize: 128
+isolation: TRANSACTION_REPEATABLE_READ
+loaderthreads: 1
+terminals: 1
+collect_pg_stat_statements: true
+use_dist_in_explain : true
+works:
+    work:
+        time_secs: 120
+        active_terminals: 1
+        rate: unlimited
+        warmup: 30
+microbenchmark:
+    class: com.oltpbenchmark.benchmarks.featurebench.customworkload.YBDefaultMicroBenchmark
+    properties:
+        setAutoCommit: true
+        create:
+            - DROP TABLE IF EXISTS pkey_for_bitmapscan_tbl_1;
+            - CREATE TABLE pkey_for_bitmapscan_tbl_1(col_bigint_id_1 bigint, col_bigint_1 bigint, col_bigint_2 bigint, col_bigint_3 bigint, col_bigint_4 bigint, col_bigint_5 bigint, col_float2_1 float(2), col_float2_2 float(2), col_float5_1 float(5), col_float5_2 float(5), col_boolean_1 boolean, col_varchar10_1 varchar(10), col_varchar100_1 varchar(100), col_varchar100_2 varchar(100), col_varchar500_1 varchar(500), PRIMARY KEY(col_bigint_id_1 ASC));
```

Comment:
Do you intend to create a range partitioned table? I see primary key as range partitioned.

Reply:
I used the same table definition that ScanG4 used. I'm not sure what impact that would have on performance, bitmap scans will work either way. Do you think I should change it? HASH probably matches the common use case a bit more.

Reply:
Yes. Range partitioned tables will any how get covered under colocated tables so it is better to cover hash partitioned tables for YB regular.